### PR TITLE
Resolves #111 - Setup Newrelic

### LINKED ISSUE #111: Setup newrelic

### PR DESCRIPTION
**Pull request by mathieugagne**
Resolves #111 - Setup Newrelic

This is simply the configuration part. To test and ensure all is well we'd need to deploy to production and create some traffic. Considering the app is in maintenance mode, we could leave this to later.

Standard integration on production, nothing particular.

### CODE REVIEW

File: config/newrelic.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Generated December 05, 2015
+# 
+# This configuration file is custom generated for app30169872@heroku.com
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: 37035e8b605929eec1e27b4958b89248af195522
```
Comment (benichu) [inline config/newrelic.yml]: should this be committed? or defined as a heroku ENV variable?

Comment (mathieugagne) [inline config/newrelic.yml]: Yikes! You're 100% right. Not that this API key grants much power but I'll do so

Comment (cawel) [inline config/newrelic.yml]: @mathieugagne and since this PR is world-visible, you should also probably generate a new `license_key` value !

**Comment by mathieugagne** [comment]
- Rebased (in case)
- Renewed license key
- Using ENV variable for Newrelic API key

File: config/newrelic.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  # agent_enabled: false
+
+  # Logging level for log/newrelic_agent.log
+  log_level: info
+
+
+# Environment-specific settings are in this section.
+# RAILS_ENV or RACK_ENV (as appropriate) is used to determine the environment.
+# If your application has other named environments, configure them here.
+development:
```
Comment (benichu) [inline config/newrelic.yml]: should this be defined as no contributors will have access to the `NEW_RELIC_LICENSE_KEY` (except maintainers with access to Heroku, and if so, we need to have a section in the README to explain the setup process)

File: config/newrelic.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  # NOTE: There is substantial overhead when running in developer mode.
+  # Do not use for production or load testing.
+  developer_mode: true
+
+test:
+  <<: *default_settings
+  # It doesn't make sense to report to New Relic from automated test runs.
+  monitor_mode: false
+
+staging:
```
Comment (benichu) [inline config/newrelic.yml]: We have no `staging` environment

**Comment by benichu** [comment]
When you are done with my questions/comments, you can squash your commits into one single descriptive one, and I'll merge.

**Comment by mathieugagne** [comment]
Squashed and ready to go. :racehorse:

**Comment by benichu** [comment]
I guess we keep the staging and development sections in `config/newrelic.yml`?

File: config/newrelic.yml (hunk 2 above)
Comment (mathieugagne) [inline config/newrelic.yml]: This is only active and needed on the production environment. No need I think

File: config/newrelic.yml (hunk 3 above)
Comment (mathieugagne) [inline config/newrelic.yml]: You're right. Could add confusion.

**Comment by mathieugagne** [comment]
Sorry about that, comments weren't there for me at the time

**Pull request merged**